test/e2e/perf/logging_soak.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package e2e
+package perf
```

Comment:
The logging sat test is specifically a performance test to vet logging, so I dunno.  Seems more performance oriented then logging behavioral testing.